packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.base.tsx

```diff
@@ -13,6 +13,7 @@ export interface IScrollablePaneContext {
     updateStickyRefHeights: () => void;
     sortSticky: (sticky: Sticky) => void;
     notifySubscribers: (sort?: boolean) => void;
+    getScrollPosition: () => number;
```

Comment:
I'm not sure we need to add this to the context if you're only using it in the `_onScrollEvent` function for `Sticky`.  We already have access to the `container`, so you can just use `container.scrollTop` to get the scroll position.